composer.json

```diff
@@ -37,7 +37,7 @@
         "php-http/guzzle6-adapter": "^1.1.1",
         "php-http/react-adapter": "^0.2.1",
         "php-http/buzz-adapter": "^0.3",
-        "symfony/symfony": "^2.8 || ^3.0",
```

Comment:
according to travis-ci, we have either a hard dependency on TwigBundle or its at least optional and needed for testing. if its optional we should add it to the (not yet existing) `suggest` section along with an explanation what its needed for.